content/en/real_user_monitoring/guide/monitor-hybrid-react-native-applications.md

```diff
@@ -0,0 +1,323 @@
+---
+title: Monitor Hybrid React Native Applications
+kind: guide
+description: Guide for monitoring hybrid React Native applications.
+further_reading:
+- link: '/monitors/create/types/real_user_monitoring/'
+  tag: 'Documentation'
+  text: 'Learn about RUM Monitors'
+---
+
+## Overview
+
+If you have a hybrid application built on React Native, you can use Datadog to monitor the same application both from the React Native and native sides.
+
+RUM Events are reported as coming from the same application and the same source in Datadog RUM.
+
+## Limitations
+
+- For **errors, resources, and interactions tracking**, the SDKs can work in two ways:
+  - Through *auto-instrumentation* - Some React classes and methods are modified to automate this. Auto-instrumentation for JavaScript errors, resources, and interactions can only be started from JavaScript code.
+  - Through *manual instrumentation* - For example, if you want to report something you consider an error but is not going to crash the app. 
+- You can share the same instance of the core SDK between native and React Native. This allows you to initialize the native SDK on either the native side or on the React Native side (by calling `DdSdkReactNative.initialize`) and have it initialized for both sides, with events appearing in the same RUM session. React Native uses the default core instance. This means that you can use *manual instrumentation* on both sides, but *auto-instrumentation* is only activated for the side that the SDK was initialized.
```

Comment:
Does this mean that you initialize it once and it's done for both without having to initialize it for the other, or is this trying to say that you can initialize it on both sides?

Reply:
clarified - thanks!